packages/dd-trace/test/config.spec.js

```diff
@@ -1979,6 +2010,36 @@ describe('Config', () => {
     })
   })
 
+  context('llmobs config', () => {
+    it('should disable llmobs by default', () => {
+      const config = new Config()
+      expect(config.llmobs.enabled).to.be.false
+    })
+
+    it('should enable llmobs if DD_LLMOBS_ENABLED is set to true', () => {
+      process.env.DD_LLMOBS_ENABLED = 'true'
+      const config = new Config()
+      expect(config.llmobs.enabled).to.be.true
+    })
+
+    it('should disable llmobs if DD_LLMOBS_ENABLED is set to false', () => {
+      process.env.DD_LLMOBS_ENABLED = 'false'
+      const config = new Config()
+      expect(config.llmobs.enabled).to.be.false
+    })
+
+    it('should enable llmobs with options and DD_LLMOBS_ENABLED is not set', () => {
+      const config = new Config({ llmobs: {} })
+      expect(config.llmobs.enabled).to.be.true
+    })
+
+    it('should have DD_LLMOBS_ENABLED take priority over options', () => {
```

Comment:
Is this expected? Env vars take precedence over programmatically set values?

Reply:
Normally no, but I believe this is how we do it in the Python SDK, right? `DD_LLMOBS_ENABLED=false` overrides `LLMObs.enable()` (which, here, is the same as just setting the `llmobs` option alltogether).

Reply:
Ahh yes, thanks for reminding me 🤦 this is fine, just wanted to double check. Thanks